src/Components/Navbar.js

```diff
@@ -0,0 +1,44 @@
+import React, {useState} from 'react'
+import './Navbar.css'
+import logo from './images/logo.png'
+import heart from './images/heart.png'
+import personIcon from './images/personIcon.png'
+
+
+const Navbar = () => {
+
+    const [menuToggle, setMenuToggle] = useState(false);
+
+    return (
+        <>
+            <div className="navbarBackground">
+                <img id="logo" src={logo} alt="logo"/>
+                <div className={menuToggle ? "linksTab" : "undisplayed"}>
+                    <p>Покупателей\собственников онлайн: 5</p>
+                    <div className="wrapper">
+                        <div className="navLinks">
+                            <a href="#">Как купить?</a>
+                            <a href="#">Как продать?</a>
+                            <a href="#">Новости</a>
+                            <a href="#">О проекте</a>
+                        </div>
+                        <div className="personalLinks">
+                            <img src={heart} alt="heart"/>
+                            <a href="#">Избранное</a>
+                            <img src={personIcon} alt="personIcon"/>
+                            <a href="#">Личный кабинет</a>
+                        </div>
+                    </div>
+                </div>
+                <div className="menuBurger" onClick={() => setMenuToggle(!menuToggle)}>
```

Comment:
Cand faci setMenuToggle(!menuToggle) nu e garantat ca valoarea la menuToggle va fi cea mai ultima, pentru ca sa ne asiguram ca valoarea care o schimbam e curenta(up to date) putem face ceva de genu: setState((prev) => !prev). incearca sa faci asa